src/plugins/shared_ux/public/components/empty_state/no_data_views/no_data_views_component.tsx

```diff
@@ -9,29 +9,17 @@
 import React, { lazy, Suspense } from 'react';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n-react';
-import {
-  EuiButton,
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiLoadingSpinner,
-  EuiSpacer,
-  useEuiTheme,
-} from '@elastic/eui';
+import { EuiButton, EuiEmptyPrompt, EuiLoadingSpinner, EuiEmptyPromptProps } from '@elastic/eui';
 import { css } from '@emotion/react';
-import { EuiText } from '@elastic/eui';
-import { EuiPageContent } from '@elastic/eui';
 import { DocumentationLink } from './documentation_link';
-/** This is an exception to include a sass file.
- * Current styling requires `breakpoint` mixin, which has not been moved to emotion yet
- */
-import './no_data_views_component.scss';
 
 const Illustration = lazy(() => import('../assets/data_view_illustration'));
 
 interface Props {
   canCreateNewDataView: boolean;
   onClickCreate?: () => void;
   dataViewsDocLink?: string;
+  emptyPromptColor?: EuiEmptyPromptProps['color'];
```

Comment:
I've added this option to customize the background color/display of the EuiEmptyPrompt, because it's very dependent on it's location.